client/build/get_build_parameters.mjs

```diff
@@ -14,7 +14,7 @@
 
 import minimist from 'minimist';
 
-const VALID_PLATFORMS = ['linux', 'windows', 'ios', 'macos', 'maccatalyst', 'android', 'browser'];
+const VALID_PLATFORMS = ['linux', 'linux_arm64', 'windows', 'ios', 'macos', 'maccatalyst', 'android', 'browser'];
```

Comment:
I would suggest to keep the platform as it is, and maybe add a new `arch` parameter?

Reply:
done